### DISCUSSION TITLE
Different kinds of balloons?

### DISCUSSION THREAD

**Discussion by jdclifton2** [nathanhoad/godot_dialogue_manager#249] [Q&A] [answered]
Hi,

What is the best way to create different kinds of balloons for different characters? For example, I have one NPC that I want to have a pink dialogue box, and another i want to have a brown dialogue box. How should I go about doing this? I tried using the tool to create another generic instance, and then change that generic instance to what I wanted but this did not work the way I expected. In fact, the first NPC I spoke to determined how the balloon for the other NPC would look. 

My current set up is to have my actionable an exported variable for the balloon, but how do I create these new balloons? 

Thanks

**Comment by nathanhoad** [comment]
One way would be to have a dictionary of names that maps to colours in your custom balloon. Then you can read the character name from the dialogue line and set the balloon colour from the dictionary.

**Reply by jdclifton2** [reply, marked as answer]
Thanks for the response.

If anyone else is curious, this is covered in this video: https://youtu.be/hKQ_s5tl4dI?feature=shared
